mirage/config.js

```diff
@@ -1,6 +1,80 @@
 import { okapi } from 'stripes-config'; // eslint-disable-line import/no-unresolved
 import { Response } from '@bigtest/mirage';
 
+/**
```

Comment:
maybe we should yank these out into a `mirage/helpers.js` file?

Reply:
@sadatay - do you mean the nameCompare, relevanceCompare and getQuery functions? Also eventually an includesWords?

Reply:
Including `searchRouteFor` and `nestedResourceRouteFor` these helpers are getting up there in numbers. We should move those over to a helpers file too with the rest. So besides imports, the first thing in this file would be `export default function configure()`

Reply:
Added helpers file, moved over noted functions and added new includesWords function as suggested. Thanks for suggestions!